code/game/objects/items/weapons/RCL.dm

```diff
@@ -24,18 +24,38 @@
 	if(istype(W,/obj/item/stack/cable_coil))
 		if(!loaded)
 			loaded = W
+			loaded.max_amount = 90 //We store a lot.
 			user.drop_item(W,src)
 		else
 			loaded.preattack(W,user,1)
 		update_icon()
 		user << "<span class='notice'>You add the cables to the [src]. It now contains [loaded.amount].</span>"
+	else if(isscrewdriver(W))
+		if(!loaded) return
+		user << "<span class='notice'>You loosen the securing screws on the side, allowing you to lower the guiding edge and retrieve the wires.</span>"
+		if(loaded.amount>60) //Dump the first coil.

```

Comment:
All you have to do is
while(loaded >= 0)
And and then check that it's greater than 30.